doc/architecture/002-person-model.md

```diff
@@ -0,0 +1,29 @@
+---
+Date: 2024-06-01
+Topic: Person Model
+Attendance: Ben, Mae, Justin, Chris, Ken
+Status: accepted
+---
+
+# Person Model
+
+## Context
+
+The current application is using `AdopterFosterAccount` and `StaffAccount`, but they are not actually holding useful information and mostly require developers to forward through it to reach information they actually want from the current user, like the information stored in the `AdopterFosterProfile`.
+
+We are abandoning the two account models as well as `AdopterFosterProfile` and `OrganizationProfile` and in their place using a `Person` model.
```

Comment:
For clarity, I believe `OrganizationProfile` columns were going to be added back to `Organization`.